src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,41 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    width,
+    height,
+    content,
```

Comment:
If it's image url, i suggest a more descriptive name, like imageUrl

Reply:
Fixed